test/views/declaration/OfficeOfExitSupplementaryViewSpec.scala

```diff
@@ -79,6 +80,12 @@ class OfficeOfExitSupplementaryViewSpec extends UnitViewSpec with ExportsTestDat
       val saveAndReturnButton = view.getElementById("submit_and_return")
       saveAndReturnButton.text() mustBe "site.save_and_come_back_later"
     }
+
+    "display Tariff section text" in {
+      val tariffText = view.getElementsByClass("govuk-details__summary-text").first().text()
+      tariffText.text() must be("site.details.summary_text_these")
```

Comment:
Just to be consistent with rest of the test cases, we preffer to use `mustBe` not `must be`.

Similar, it's not a mistake, but always we're trying to keep one approach to be consistent

Reply:
Fixed